### ISSUE TITLE
pose net and depth net

### ISSUE THREAD

**Issue by shixinsong** [JiawangBian/SC-SfMLearner-Release#54]
Hi,
   After reading the code, I found that the posenet and depthnet networks and the loss function are consistent.It seems that the training data set is not consistent.  The two data sets can look like consecutive pictures based on time, and there seems to be no difference.  What is the specific difference between posenet and depthnet in the training process?  I have just studied the content of monocular depth estimation and hope to get your reply.
Thanks.
